### PR TITLE
Additional function names and Unknowns

### PR DESCRIPTION
**Pull request by Zintixx**
Some additional function names and resolving unknowns of a few opcodes

### CODE REVIEW

File: Inbound/0x0055.py [hunk 1]
```diff
@@ -50,11 +51,11 @@ def decode_mail_info():
...
+    add_long("Mesos")
+    add_long("LastPurchaseTimestamp?")
+    add_long("BlueMerets")
     add_long("Unknown")
-    add_long("Unknown")
-    add_long("Unknown")
-    add_long("Unknown")
-    add_long("Unknown")
+    add_long("RedMerets")
     add_long("Unknown")
```
Comment (kOchirasu) [inline Inbound/0x0055.py:59]: I wonder if these are also meret values, there's like 5 different meret types.

File: Inbound/0x006C.py [hunk 2]
```diff
@@ -32,14 +32,14 @@
...
             add_unicode_str("PlotName")
             add_long("AccountId")
-elif f == 3: # House Plot related
+elif f == 3: # House Plot expiration
     count = add_int("count")
     for i in range(count):
         with Node("SaleStateData " + str(i)):
             add_int("PlotId")
-            add_int("Unknown")
+            add_int("ApartmentNumber")
```
Comment (kOchirasu) [inline Inbound/0x006C.py:43]: Can you also update this in `0x006B`

File: Inbound/0x0055.py (hunk 1 above)
Comment (Zintixx) [reply Inbound/0x0055.py:59]: Just tried it out. Doesn't look like it is.

Comment (kOchirasu) [reply Inbound/0x0055.py:59]: thanks for confirming

**Pull request merged**